src/main.py

```diff
@@ -80,18 +82,15 @@ def main():
     # get command line args
     args = cli.parse_cli_args()
 
-    # print canteens
+    # print locations
     if args.locations:
-        with open("canteens.json", "r", encoding="utf-8") as canteens:
-            print(json.dumps(json.load(canteens)))
+        with open("canteens.json", "r", encoding="utf-8") as locations:
```

Comment:
Shouldn't this be generated through the new enums?

Reply:
:+1: